app/controllers/application_controller.rb

```diff
@@ -1,4 +1,9 @@
+require 'application_responder'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.